client/src/components/Overview/AddToCart.jsx

```diff
@@ -30,16 +30,15 @@ function AddToCart(props) {
       return;
     }
 
-    const quantity = Math.min(styles[styleSelector].skus[sizeSelection].quantity, 15);
-
+    const quantity = styles[styleSelector].skus[sizeSelection] && Math.min(styles[styleSelector].skus[sizeSelection].quantity, 15);
```

Comment:
Because of the way this is written, `quantity` will either be a number (the minimum of 15 and whatever is in `skus[sizeSelection].quantity`) **OR** it will be `false` (when `skus[sizeSelection].quantity` is `undefined`).

Line 35: `_.range(quantity + 1)` would then *always* map at least once, since `false + 1` evaluates to `1`.

Is this intended behavior?

Reply:
yes. the app crashes if it isn't done this way